src/helpers/xcode.ts

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs'
+
+import { info, UploadLogger } from '../helpers/logger'
+import {
+  XcodeCoverageFileReport,
+  XcodeCoverageReport,
+} from '../types'
+import { isProgramInstalled, runExternalProgram } from "./util"
+
+export async function generateXcodeCoverageFiles(archivePath: string[]): Promise<string> {
+  if (!isProgramInstalled('xcrun')) {
+      throw new Error('xcrun is not installed, cannot process files')
+  }
+  info('Running xcode coversion...')
+
+  const coverage: XcodeCoverageReport = {}
+  const report = { coverage: coverage }
+  const path = archivePath[0]
```

Comment:
Only one path from `archivePath` is ever used. Is this intentional?

Reply:
It comes from https://github.com/codecov/uploader/pull/715/files/89e3f7d4c126d68852a2292e70a99694dbdf913f#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R174, and I was struggling to wrangle the types here.

Not sure if there's a better way to do this